db/schema.rb

```diff
@@ -154,7 +154,7 @@
     t.string "name", null: false
     t.text "comment"
     t.string "owners"
-    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true
+    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true, length: 191
```

Comment:
where does this arbitrary limit come from?

Reply:
bug in 9fc22d6254 which reformatted the schema

Reply:
https://github.com/zendesk/samson/pull/3656 to clean things up more